### DISCUSSION TITLE
[2088570]Tiny Rogues(小小肉鸽)[仅本体]

### DISCUSSION THREAD

**Discussion by NTBLing** [pjy612/SteamManifestCache#510] [Share]
` appid:2088570` 
Keys:
``` 
2088571;122e3270335372157a11a595038524e4cd4bfd7b2b9851c40ca5c25be3f85914
```
[2088571_7842365050075519737.zip](https://github.com/pjy612/SteamManifestCache/files/14368662/2088571_7842365050075519737.zip)

code:F32A8A54051A2C23
邮箱:1625912198@qq.com

**Comment by CSSZYF** [comment]
#26 格式

**Reply by NTBLing** [reply]
已改
顺便问下，1.0.1更新到1.0.2，之前的内测code被现在的强制覆盖，这正常吗

**Reply by pjy612** [reply]
* 密钥格式 #26 
* code 无视

**Reply by NTBLing** [reply]
我的
已改

**Reply by pjy612** [reply]
emmmm
提示 密钥后面 没显示复制按钮
另外 你以前分享的格式都是对的，怎么现在就懈怠了呢~

**Reply by NTBLing** [reply]
想着自己都发过没问题的了，再发新的不是随随便便嘛（笑哭）
